armi/reactor/blueprints/__init__.py

```diff
@@ -572,20 +572,19 @@ def migrate(bp: Blueprints, cs):
     """
     Apply migrations to the input structure.
 
-    This is a good place to perform migrations that address changes to the system design
-    description (settings, blueprints, geom file). We have access to all three here, so
-    we can even move stuff between files. Namely, this:
-
-     * creates a grid blueprint to represent the core layout from the old ``geomFile``
-       setting, and applies that grid to a ``core`` system.
-     * moves the radial and azimuthal submesh values from the ``geomFile`` to the
-       assembly designs, but only if they are uniform (this is limiting, but could be
-       made more sophisticated in the future, if there is need)
-
-    This allows settings-driven core map to still be used for backwards compatibility.
-    At some point once the input stabilizes, we may wish to move this out to the
-    dedicated migration portion of the code, and not perform the migration so
-    implicitly.
+    This is a good place to perform migrations that address changes to the system design description
+    (settings, blueprints, geom file). We have access to all three here, so we can even move stuff
+    between files. Namely, this:
+
+     * creates a grid blueprint to represent the core layout from the old ``geomFile`` setting, and
+       applies that grid to a ``core`` system.
+     * moves the radial and azimuthal submesh values from the ``geomFile`` to the assembly designs,
+       but only if they are uniform (this is limiting, but could be made more sophisticated in the
+       future, if there is need)
+
+    This allows settings-driven core map to still be used for backwards compatibility. At some point
+    once the input stabilizes, we may wish to move this out to the dedicated migration portion of
+    the code, and not perform the migration so implicitly.
```

Comment:
Is there any actual change here, or did you just change the length of all the lines?

Reply:
I reverted this change back to `main`. 

It's gone. I was doing a lot of work in this file, but then reverted that work and left the comment change.  It's back to normal now.